target/linux/realtek/dts-5.10/rtl838x.dtsi

```diff
@@ -62,17 +62,9 @@
 
 	compatible = "realtek,rtl838x-soc";
 
-	osc: oscillator {
-		compatible = "fixed-clock";
-		#clock-cells = <0>;
-		clock-frequency = <25000000>;
-	};
-
 	ccu: clock-controller {
 		compatible = "realtek,rtl8380-clock";
 		#clock-cells = <1>;
-		clocks = <&osc>;
-		clock-names = "ref_clk";
```

Comment:
I don't think being able to omit this, is worth the extra boilerplate in the clock driver.

Reply:
This is only needed if oscillator stays in DT as fixed clock